lua/obsidian/note.lua

```diff
@@ -209,6 +209,9 @@ note.from_lines = function(lines, path, root)
   if #frontmatter_lines > 0 then
     local frontmatter = table.concat(frontmatter_lines, "\n")
     local ok, data = pcall(yaml.loads, frontmatter)
+    if type(data) == 'string' then
+      data = {data = nil}
```

Comment:
How about just this:
```suggestion
      data = {}
```